fivox/eventValueSummationImageSource.hxx

```diff
@@ -25,6 +25,8 @@
 
 #include <itkProgressReporter.h>
 
+#include <lunchbox/clock.h>
```

Comment:
Nothing, but if we decide to eventually remove clock, this is one spot less that would require changes.